roles/contiv_network/tasks/main.yml

```diff
@@ -20,21 +20,35 @@
 - name: download netmaster and netplugin
   get_url:
     validate_certs: "{{ validate_certs }}"
-    url: https://github.com/contiv/netplugin/releases/download/v0.0.0-12-11-2015.20-54-40.UTC/netplugin-v0.0.0-12-11-2015.20-54-40.UTC.tar.bz2
+    url: https://github.com/contiv/netplugin/releases/download/v0.1-01-28-2016.03-55-05.UTC/netplugin-v0.1-01-28-2016.03-55-05.UTC.tar.bz2
     dest: /tmp/contivnet.tar.bz2
 
+- name: ensure netplugin directory exists
+  file: path=/usr/bin/contiv/netplugin state=directory
+
 - name: install netmaster and netplugin
   shell: tar vxjf /tmp/contivnet.tar.bz2
   args:
-    chdir: /usr/bin/
+    chdir: /usr/bin/contiv/netplugin
```

Comment:
@DivyaVavili as per the discussion with @mapuri, yes it is to better organise our binaries/files. This will also eases up the cleanup.